lib/ReactViews/StandardUserInterface/StandardUserInterface.jsx

```diff
@@ -247,7 +248,7 @@ const StandardUserInterface = observer(
         !this.shouldUseMobileInterface();
       const showStoryPanel =
         this.props.terria.configParameters.storyEnabled &&
-        this.props.terria.stories.length &&
+        this.props.terria.stories.length > 0 &&
```

Comment:
This check caused `showStoryPanel` to be 0 if the length is 0 and when used in JSX for conditional rendering it printed a stray 0 in the dom.